nexus/tests/output/cmd-nexus-noargs-stderr

```diff
@@ -1,6 +1,13 @@
-error: the following required arguments were not provided:
-  <CONFIG_FILE_PATH>
+See README.adoc for more information
 
-Usage: nexus <CONFIG_FILE_PATH>
+Usage: nexus [OPTIONS] [CONFIG_FILE_PATH]
```

Comment:
I think what is supported is:
`nexus -i`
or
`nexus -o`
or
`nexus <config_file_path>`

I don't know how to make Clap enforce that though, and what you have here essentially does the same
thing anyway.

Reply:
Frankly, I'd like to change nexus to operate like `nexus run <cfg>` or `nexus openapi` like all (?) of the other openapi servers in omicron.  Updating this was a stop-gap so I could get it to emit openapi docs for me without having to rustle up a valid configuration.